src/shared/varlink-io.systemd.udev.c

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include "varlink-io.systemd.udev.h"
+
+static SD_VARLINK_DEFINE_METHOD(StartExecQueue);
+
+static SD_VARLINK_DEFINE_METHOD(StopExecQueue);
+
+static SD_VARLINK_DEFINE_METHOD(
+                SetEnvironment,
+                SD_VARLINK_DEFINE_INPUT(assignments, SD_VARLINK_STRING, SD_VARLINK_ARRAY));
+
+static SD_VARLINK_DEFINE_METHOD(
+                UnsetEnvironment,
+                SD_VARLINK_DEFINE_INPUT(names, SD_VARLINK_STRING, SD_VARLINK_ARRAY));
+
+static SD_VARLINK_DEFINE_METHOD(
+                SetChildrenMax,
+                SD_VARLINK_DEFINE_INPUT(n, SD_VARLINK_INT, 0));
+
+static SD_VARLINK_DEFINE_METHOD(Exit);
```

Comment:
I agree that it's time to deprecate `--exit`. It's nothing of use in systemd world anyway.